scripts/install.sh

```diff
@@ -855,6 +856,14 @@ function get_binary_from_url() {
     fi
 }
 
+function set_acl_on_log_paths() {
+for path in ${LOG_FILE_PATHS}; do
+    echo -e "Running: setfacl -R -m d:u:${SYSTEM_USER}:r-x,u:${SYSTEM_USER}:r-x,g:${SYSTEM_USER}:r-x ${path}"
+    setfacl -R -m d:u:${SYSTEM_USER}:r-x,u:${SYSTEM_USER}:r-x,g:${SYSTEM_USER}:r-x ${path}
```

Comment:
is `setfacl` command always installed? If not please handle scenario when its missing

Reply:
setfacl command conditionally tested for in latest commit